micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ChangeLanguage.js

```
@@ -18,12 +22,13 @@ const ChangeLanguage = (prop) => {
     return (
       <div>
         <Dropdown
+          className={"language-dropdown"}
           option={languages}
-          selected={languages.find((language) => language.value === selectedLanguage)}
+          selected={languages?.find((language) => language?.value === selectedLanguage)}
           optionKey={"label"}
           select={handleChangeLanguage}
           freeze={true}
-          customSelector={<label className="cp">{languages.find((language) => language.value === selected).label}</label>}
+          customSelector={<label className="cp">{t(languages?.find((language) => language?.value === selected)?.label)}</label>}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)